pkg/cloudprovider/provider/kubevirt/types/types.go

```diff
@@ -75,7 +76,9 @@ type Disk struct {
 
 // Affinity.
 type Affinity struct {
-	PodAffinityPreset     providerconfigtypes.ConfigVarString `json:"podAffinityPreset,omitempty"`
+	// Deprecated
+	PodAffinityPreset providerconfigtypes.ConfigVarString `json:"podAffinityPreset,omitempty"`
```

Comment:
please add explanation why it's deprecated now. Should be
`// Deprecated: <description>`

We deprecate that field because KubeVirt supports now topology spread constraints.